src/txmempool.h

```diff
@@ -479,21 +479,21 @@ class CTxMemPool
 protected:
     const int m_check_ratio; //!< Value n means that 1 times in n we check.
     std::atomic<unsigned int> nTransactionsUpdated{0}; //!< Used by getblocktemplate to trigger CreateNewBlock() invocation
-    CBlockPolicyEstimator* minerPolicyEstimator;
+    CBlockPolicyEstimator* minerPolicyEstimator GUARDED_BY(cs);
```

Comment:
This is only ever modified by assigning it in the constructor, can be `const` like `m_check_ratio`.
```suggestion
    CBlockPolicyEstimator* const minerPolicyEstimator;
```

Reply:
And the object protects itself via its own mutex. Done.